inc/saiport.h

```diff
@@ -292,6 +292,57 @@ typedef enum _sai_port_prbs_config_t
     SAI_PORT_PRBS_CONFIG_ENABLE_TX
 } sai_port_prbs_config_t;
 
+/**
+ * @brief Attribute data for #SAI_PORT_ATTR_FAIL_OVER_MODE
+ * Used for Fail over mode configuration on port
+ */
+typedef enum _sai_port_fail_over_mode_t
+{
+    /** Fail over mode disable */
+    SAI_PORT_FAIL_OVER_MODE_DISABLE,
+
+    /** Enable Fail over mode on the port */
+    SAI_PORT_FAIL_OVER_MODE_ENABLE,
```

Comment:
The enum value SAI_PORT_FAIL_OVER_MODE_ENABLE seems redundant. If fail-over is enabled on one port, then it has to be either primary or secondary.